### ISSUE TITLE
Automated translation

### ISSUE THREAD

**Issue by axel584** [hlostam/srt2anki#3]
I I added an automatic translation of the created files (using deeptranslator).
Would you like a Pull Request?

I am also learning Italian, but I am French speaking

**Comment by hlostam** [comment]
Yes please and thanks for spotting the issues and suggesting the pull requests., Sorry I did not receive any notification of these issues and was not checking the repo so found out about this by accident!

**Comment by hlostam** [comment]
Yes please and thanks for spotting the issues and suggesting the pull requests., Sorry I did not receive any notification of these issues and was not checking the repo so found out about this by accident!
